### PR TITLE
Remove argumentless version of :unbind

### LINKED ISSUE #165: Remove argumentless version of :unbind
Derived from https://github.com/rexim/Morganey/pull/163#issuecomment-244334364

### PR DESCRIPTION
**Pull request by rexim**
Close #165

### CODE REVIEW

File: src/main/scala/me/rexim/morganey/Commands.scala [hunk 1]
```diff
@@ -61,6 +60,8 @@ object Commands {
         val message = removed match {
           case Nil       => "No bindings were removed!"
           case hd :: Nil => s"Binding '$hd' was removed!"
+          case bindings if bindings.size > unbindCountThreshold =>
+            s"${bindings.size} bindings were removed!"
           case in :+ ls  =>
             val bindingEnumeration = in.map(b => s"'$b'").mkString(", ") + s" and '$ls'"
```
Comment (ForNeVeR) [inline src/main/scala/me/rexim/morganey/Commands.scala]: A nitpick: there should be a trailing comma here (so-called [_Oxford comma_](https://en.wikipedia.org/wiki/Serial_comma)) before `and`.

**Comment by ForNeVeR** [comment]
LGTM :+1:

File: src/main/scala/me/rexim/morganey/Commands.scala (hunk 1 above)
Comment (rexim) [reply src/main/scala/me/rexim/morganey/Commands.scala]: Thanks! Done.

**Comment by keddelzz** [comment]
LGTM :+1:

**Comment by rexim** [comment]
Cool! Thank you guys for the review!

**Pull request merged**